Source/artifacts/CommandInquirer.js

```diff
@@ -24,43 +24,24 @@ export class CommandInquirer {
     }
     /**
      * Gets the inquirer.js prompt answers based on the language
-     * @param {String} language
+     * @param {any} flags
      * @returns {Promise<any>} The answers
      */
-    promptUser(language){
-        if (language === 'csharp') {
-            return this._getCSharpPrompt()
+    promptUser(flags){
+        if (flags.language === 'csharp') {
```

Comment:
Lets figure out a way to not have to have an if/else pattern for adding new languages. Relates to #34. 
We need something like an interface for the things that we want to provide language specific things for and register every language specific implementation at startup. We could then have a context object which would then hold a reference to instances of these. 

Btw: Flags -> context